models/User.js

```diff
@@ -0,0 +1,24 @@
+const mongoose = require("mongoose");
+
+const userSchema = new mongoose.Schema(
+  {
+    name: {
+      type: String,
+      required: "name is a required field"
+    },
+
+    email: {
```

Comment:
You could add a validator to check for email syntax. Although we have client side validation in place, it is a good practice to also have Server side validation